tests/longevity/longevity_test.go

```diff
@@ -52,39 +51,39 @@ var _ = BeforeSuite(func() {
 	InitInstance()
 	populateIntervals()
 	populateDisruptiveTriggers()
-	err := backup.UpdatePxBackupAdminSecret()
-	Expect(err).NotTo(HaveOccurred())
+	//err := backup.UpdatePxBackupAdminSecret()
```

Comment:
Remove code

Reply:
Hi Vinayak,
these are px-backup changes and needs some additional changes for it work along with regular longevity.
Hence commented for time being and will be fixed in next sprint